main.go

```diff
@@ -2,10 +2,10 @@ package main
 
 import (
 	"github.com/gorilla/mux"
-	. "github.com/mickael-kerjean/filestash/server/common"
-	. "github.com/mickael-kerjean/filestash/server/ctrl"
-	. "github.com/mickael-kerjean/filestash/server/middleware"
-	_ "github.com/mickael-kerjean/filestash/server/plugin"
+	. "github.com/mickael-kerjean/filestash/src/common"
```

Comment:
I much prefer the current folder structure with both a `client` folder for everything client related and a `server` folder for everything server related

Reply:
server folder is not the standar form for go project, and it doesn't work with loca copy with server folder.